sandbox.go

```diff
@@ -719,6 +725,34 @@ func (sb *sandbox) SetKey(basePath string) error {
 	return nil
 }
 
+func (sb *sandbox) EnableService() error {
+	for _, ep := range sb.getConnectedEndpoints() {
+		if !ep.serviceEnabled {
```

Comment:
This should be lock protected and may be even set within the lock. We can unset if there is a failure.